### ISSUE TITLE
not working period for me. 

### ISSUE THREAD

**Issue by lukejclark** [sorccu/cufon#195]
I am using font Caecilia Com and it will not work period on any browser. What is going wrong? Please help! http://clarkitdesign.com/test/

<link rel="stylesheet" type="text/css" href="css/style.css" /> 
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.3.2/jquery.min.js" type="text/javascript"></script>
    <script src="http://clarkitdesign.com/test/cufon-yui.js" type="text/javascript"></script>
    <script type="text/javascript" src="http://clarkitdesign.com/test/js/Caecilia_Com_55_Roman_400-Caecilia_Com_45_Light_700.font.js"></script> 

.title-large {
    color: #deb31c; 
    font-family: 'Caecilia Com';
    font-size: 30px;
    font-weight: 700;
    text-align: left;
    padding-bottom: 25px;
}
